pages/404.tsx

```diff
@@ -0,0 +1,16 @@
+// 404.js
+
+import Link from "next/link";
+
+export default function FourOhFour() {
+  return (
+    <>
+      <h1>404 - Page Not Found</h1>
+      We&apos;re very sorry - we could not find the page you were looking for.
+      Please navigate to the Teleport <Link href="/docs/">
+        Documentation
+      </Link>{" "}
+      or the <a href="/">Home Page</a> to find what you&apos;re looking for.
```

Comment:
Assuming we could wrap this content with the standard page layout component (which we should be doing), we could change it to something like this:

```suggestion
      We&apos;re very sorry - we could not find the page you were looking for.
      Please navigate to the Teleport <Link href="/docs/">
        Documentation
      </Link>{" "}
      or the <a href="/">Home Page</a>, or try our search bar.
```

Otherwise:

```suggestion
      We&apos;re very sorry - we could not find this page.
      Try  the Teleport <Link href="/docs/">Documentation
      </Link>{" "}
      or the <a href="/">Home Page</a> to find what you&apos;re looking for.
```

Reply:
I'd just changed the file's extension in this PR without changing the content, so I'd propose we do it in a separate PR.

Reply:
Tracking in https://github.com/gravitational/docs/issues/240